tests/rustdoc-ui/issue-105742.rs

```diff
@@ -1,19 +1,37 @@
 // compile-flags: -Znormalize-docs
-
 use std::ops::Index;
 
 pub fn next<'a, T>(s: &'a mut dyn SVec<Item = T, Output = T>) {
+    //~^ERROR
```

Comment:
Question: Is there a nicer way to do this if you are not looking to tests the specific errors?
Or should I just write out all the errors produced here properly?

Reply:
Writing the beginning of the error message would be nice indeed. No need for more I think.

Reply:
- [ ] will do